src/test/java/org/apache/commons/compress/archivers/LongPathTest.java

```diff
@@ -142,7 +140,7 @@ public void testArchive(final File file) throws Exception {
         }
         try {
             checkArchiveContent(ais, expected);
-        } catch (final AssertionFailedError e) {
+        } catch (final Exception e) {
```

Comment:
-1: Porting to JUnit 5 means using `Assert.assertThrows()`, not whatever this is.

Reply:
I'm with you on this one (and the one in the LongSymLinkTest). Just did not want to do too much in one go.

If I understand the test correctly we can get rid of the try-catch block completely since we do not want the check to throw an exception.